divisible.rb

```diff
@@ -0,0 +1,15 @@
+#(1) Write a method that returns in an array all the numbers between 1 and 100 that are divisible by 2 or 3 or 5. 
+#Then call the method in your program and print out what it returns.  
+#Call the program divisible.rb.
+
+def divnumbers
+    array = []
+    (1..100).each do |n|
```

Comment:
looks good! #each gets the job done, but bonus: think about a refactor with #filter that might make more semantic sense.